### ISSUE TITLE
About dataset

### ISSUE THREAD

**Issue by CitySorth** [zhu-xlab/M3R-CR#12]
Why does the dataset proposed in the paper consist solely of a training set and a test set? When training the model, the validation set also utilizes the training set. This differs from the traditional 7:2:1 division of training, validation, and test sets.
